zds/forum/models.py

```diff
@@ -228,6 +228,10 @@ def meta_description(self):
             return first_post.text
         return Topic.__remove_greetings(first_post)[: settings.ZDS_APP["forum"]["description_size"]]
 
+    @property
+    def update(self):
+        return self.get_last_visible_post().update_index_date
```

Comment:
L'usage de `update_index_date` pose soucis car cette date est par exemple mise à jour lorsqu'un staff masque ou démasque un message. Je suggère d'utiliser `update` s'il y a eu une mise à jour et `pubdate` s'il n'y en a pas eu. Cela évite de détourner `update_index_date` qui est prévu pour l'indexation dans le moteur de recherche seulement. :)
```suggestion
        last_visible_post = self.get_last_visible_post()
        return last_visible_post.update or last_visible_post.pubdate
```

Reply:
Je sentais en l'écrivant que c'était un peu bancal, ta proposition me semble bien. Adoptée !